include/mgp_mock.py

```diff
@@ -0,0 +1,1916 @@
+        """
+        Check if the underlying graph is mutable.
+
+        Returns:
+            A `bool` value that represents whether the graph is mutable.
+
+        Raises:
+            InvalidContextError: If the context is not valid.
+
+        Examples:
+            ```edge.underlying_graph_is_mutable()```
+        """
+        if not self.is_valid():
+            raise InvalidContextError()
+
+        return self._vertex.underlying_graph_is_mutable()
+
+    @property
+    def id(self) -> VertexId:
+        """
+        Get the ID of the vertex.
+
+        Returns:
+            A `VertexId` representing the vertex’s ID.
+
+        Raises:
+            InvalidContextError: If vertex is out of context.
+
+        Examples:
+            ```vertex.id```
+        """
+        if not self.is_valid():
+            raise InvalidContextError()
+
+        return self._vertex.id
+
+    @property
+    def labels(self) -> typing.Tuple[Label]:
+        """
+        Get the labels of the vertex.
+
+        Returns:
+            A tuple of `Label` instances representing individual labels.
+
+        Raises:
+            InvalidContextError: If vertex is out of context.
+            DeletedObjectError: If `Vertex` has been deleted.
+
+        Examples:
+            ```vertex.labels```
+        """
+        if not self.is_valid():
+            raise InvalidContextError()
+
+        if self._vertex.is_deleted():
+            raise DeletedObjectError("Accessing deleted object.")
+
+        return tuple(Label(label) for label in self._vertex.labels)
+
+    def add_label(self, label: str) -> None:
+        """
+        Add the given label to the vertex.
+
+        Args:
+            label: The label (`str`) to be added.
+
+        Raises:
+            InvalidContextError: If `Vertex` is out of context.
+            ImmutableObjectError: If `Vertex` is immutable.
+            DeletedObjectError: If `Vertex` has been deleted.
+
+        Examples:
+            ```vertex.add_label(label)```
+        """
+        if not self.is_valid():
+            raise InvalidContextError()
+
+        if self._vertex.is_deleted():
+            raise DeletedObjectError("Accessing deleted object.")
+
+        return self._vertex.add_label(label)
+
+    def remove_label(self, label: str) -> None:
+        """
+        Remove the given label from the vertex.
+
+        Args:
+            label: The label (`str`) to be removed.
+
+        Raises:
+            InvalidContextError: If `Vertex` is out of context.
+            ImmutableObjectError: If `Vertex` is immutable.
+            DeletedObjectError: If `Vertex` has been deleted.
+
+        Examples:
+            ```vertex.remove_label(label)```
+        """
+        if not self.is_valid():
+            raise InvalidContextError()
+
+        if self._vertex.is_deleted():
+            raise DeletedObjectError("Accessing deleted object.")
+
+        return self._vertex.remove_label(label)
```

Comment:
same as above

Reply:
The `mgp_mock.remove_label` and `_mgp_mock.remove_label` methods both return `None`, so there’s no harm in doing this.
In fact, the Python API does the same thing in `mgp.py`.

Reply:
yes, yes just a matter of syntax, I agree. Then decision is up to you :)

Reply:
Since this is how it is in mgp API, we can continue this way here.